ObjectMemoryTests.c

```diff
@@ -435,16 +435,37 @@ Test(DiscoverByteLengthOfObjectWithOddSize) {
   Expect(byteLength == (size - 2)*2 - 1);
 }
 
-Test(AllocateSmallObject) {
-  ObjectPointer objectPointer, classPointer = 0x2468, reportedClass;
+Test(AllocateSmallObjectByReclaimingRightSizedSpace) {
```

Comment:
This test already existed, I've just cleaned it up and defined some values that were left undefined previously to improve repeatability.